x-pack/plugins/security_solution/server/usage/detections/detection_rule_helpers.test.ts

```diff
@@ -16,132 +15,87 @@ const createStubRule = (
   alertCount: number,
   caseCount: number
 ): DetectionRuleMetric => ({
-  rule_name: uuid(),
-  rule_id: uuid(),
+  rule_name: 'rule-name',
```

Comment:
These stubs make me a little uneasy due to upstream drift. Is there a shared mock we can use?

Reply:
I can look for one and use it if we have one. We typically end up with a few.